src/api-guides/travel/deeplink-integration-new.md

```diff
@@ -203,12 +203,54 @@ https://www.concursolutions.com/goto/hotel-shop?checkindate=2024-11-02&checkoutd
 
 This deeplink will perform a search for the hotel Travelodge LAX South in El Segundo (CA) and hotels nearby in the 5 mi default radius, with check-in date on November 2nd, 2024 and checkout date on November 3rd, 2024. It uses the Hotel Property ID from HRS as location.
 
-#### <a name="example-one-way"></a> Example: Concur Travel Internal Hotel ID location
+#### <a name="example-one-way"></a> Example: SAP Concur Travel Internal Hotel ID location
 ```
 https://www.concursolutions.com/goto/hotel-shop?checkindate=2024-11-02&checkoutdate=2024-11-03&location=1907825ad1f728ccafb22942d61a2715
 ```
 
-This deeplink will perform a search for the hotel Econo Lodge Inn & Suites in Arkansas and hotels nearby in the 5 mi default radius, with check-in date on November 2nd, 2024 and checkout date on November 3rd, 2024. It uses a Concur Travel Internal Hotel ID as location.
+This deeplink will perform a search for the hotel Econo Lodge Inn & Suites in Arkansas and hotels nearby in the 5 mi default radius, with check-in date on November 2nd, 2024 and checkout date on November 3rd, 2024. It uses a SAP Concur Travel Internal Hotel ID as location.
+
+
+## Add Hotel to existing Trip
+
+Users are able to provide a trip ID in a hotel deeplink and omit the check-in and check-out dates. Then the hotel search and subsequent hotel booking will be part of the specified trip, instead of a separate new trip. In contrast to a deeplink without trip ID, the user will end up on the hotel search page with pre-filled search criteria. The check-in and check-out dates are either taken over from the arrival of the first air booking and the departure of the last air booking of the specified trip. In case of a one-way flight the check-out date is taken over from the trip end date. Or the dates fallback to the trip start and end dates of the specified trip. These search criteria can still be edited, before a search can be started.
+Only if a hotel property ID is provided the search is automatically started.
```

Comment:
I'd mention the difference between providing a location or not earlier, also IMO simplified a bit the comment about the fallback dates.

```suggestion
Users are able to provide a trip ID in a hotel deeplink and omit the check-in and check-out dates. Then the hotel search and subsequent hotel booking will be part of the specified trip, instead of a separate new trip.

If only a trip ID is provided without a specific location, the user will end up on the hotel search page with pre-filled search criteria based on the trips booking.

If in addition to the trip ID a specific location is provided, the user will directly get to the search results based on this location. The check-in and check-out dates are taken over from the arrival of the first air booking and the departure of the last air booking of the specified trip. In case of a one-way flight the check-out date is taken over from the trip end date. If the trip contains no air booking then the trip start and end dates are used.
```

Reply:
👍 I misunderstood that the location type doesn't matter for where the user ends up (pre-filled search from vs. shop results), but the fact whether a location was provided is deciding. makes more sense